management-server/routes/referenceData.js

```diff
@@ -10,28 +10,28 @@ router.get('/', function(req, res, next) {
   res.render('index', { title: 'Veniqa Reference Data' });
 });
 
-router.get('/getCatalogBundle', passportAuth.isAuthenticated, referenceDataController.getCatalogBundle);
+router.get('/getCatalogBundle', referenceDataController.getCatalogBundle);
 
-router.get('/getStores', passportAuth.isAuthenticated, referenceDataController.getStores);
+router.get('/getStores', referenceDataController.getStores);
 
-router.get('/getRoles', passportAuth.isAuthenticated, referenceDataController.getRoles);
+router.get('/getRoles', referenceDataController.getRoles);
 
-router.get('/getWeightUnits', passportAuth.isAuthenticated, referenceDataController.getWeightUnits);
+router.get('/getWeightUnits', referenceDataController.getWeightUnits);
 
-router.get('/productCategoryList', passportAuth.isAuthenticated, passportAuth.canViewCatalog, referenceDataController.getProductCategoryList);
```

Comment:
The canView functions don't check the authenticated status. So I think if we remove this, even if their session times out, the request will still be processed.

Reply:
isAuth is being checked in `app.js`. just removing multiple middlewares in one line